backends/advanced/tests/test_conversation_models.py

```
@@ -0,0 +1,269 @@
+            model="nova-3",
+            processing_time_seconds=10.0
+        )
+
+        assert len(conversation.transcript_versions) == 1
+        assert conversation.active_transcript_version == "v1"
+        assert conversation.transcript == "Test transcript"
+        assert len(conversation.segments) == 1
+        assert version1.version_id == "v1"
+
+        # Add second transcript version without setting as active
+        version2 = conversation.add_transcript_version(
```

Ruff (0.13.1)

Local variable `version2` is assigned to but never used

Remove assignment to unused variable `version2`

(F841)